src/plugins/discover/public/application/main/components/loading_spinner/loading_spinner.tsx

```diff
@@ -6,15 +6,32 @@
  * Side Public License, v 1.
  */
 
-import './loading_spinner.scss';
-
 import React from 'react';
-import { EuiLoadingSpinner, EuiTitle, EuiSpacer } from '@elastic/eui';
+import {
+  EuiLoadingSpinner,
+  EuiTitle,
+  EuiSpacer,
+  useEuiPaddingSize,
+  useEuiBackgroundColor,
+} from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
+import { css } from '@emotion/react';
 
 export function LoadingSpinner() {
+  const loadingSpinnerCss = css`
```

Comment:
Changed to use absolute positioning since it will be overlayed on top of `DiscoverHistogramLayout` while loading.